kube/services/jobs/psql-db-backup-encrypt-job.yaml

```diff
@@ -138,6 +143,18 @@ spec:
               # Import the public key
               echo "$PUBLIC_KEY" | gpg --import
 
+              # Check and retrieve Slack webhook if not provided
+              slack=false
+              if [[ "${slackWebHook}" == 'None' || -z "${slackWebHook}" ]]; then
+                slackWebHook=$(g3kubectl get configmap global -o jsonpath={.data.slack_alarm_webhook})
+              fi
```

Comment:
We should not do kubectl operations from within the job itself, we should mount the secrets/configmaps needed and in this script just check if they're present.

Reply:
done. thanks